standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -110,8 +121,9 @@ void createCatalog(Catalog catalog)
    * @throws MetaException usually indicates a database error
    * @throws TException general thrift exception
    */
-  void alterCatalog(String catalogName, Catalog newCatalog)
-      throws NoSuchObjectException, InvalidObjectException, MetaException, TException;
+  default void alterCatalog(String catalogName, Catalog newCatalog)
```

Comment:
space before {

Reply:
should throw UnsupportedOperationException

Reply:
Done.